res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1572,60 +1810,25 @@ PioneerDDJSX.playLed = function(value, group, control) {
     PioneerDDJSX.nonPadLedControl(group, PioneerDDJSX.nonPadLeds.shiftPlay, value);
 };
 
-PioneerDDJSX.playFromCuePoint = false;
-
-PioneerDDJSX.playIllumination = function(value, group, control) {
+PioneerDDJSX.wheelLeds = function(value, group, control) {
     var deck = PioneerDDJSX.channelGroups[group],
-        cue = PioneerDDJSX.cuePressed[deck],
-        trackLoaded = 0;
+        duration = engine.getValue(group, "duration"),
+        remainingTime = duration - (value * duration),
+        speed = 0x28,
+        wheelPos = 0x01;
 
-    trackLoaded = engine.getValue(group, "track_loaded");
-    if (!trackLoaded) {
-        return;
+    if (value >= 0) {
+        wheelPos = 0x01 + ((speed * (value * duration)) % 0x48);
```

Comment:
Refer to the [Reloop Beatmix 2/4 mapping code](https://github.com/mixxxdj/mixxx/blob/master/res/controllers/Reloop-Beatmix-2-4-scripts.js#L524) for an example of how to make the code clearer.